capsicum/src/right.rs

```diff
@@ -339,11 +340,29 @@ impl FileRights {
         unsafe { libc::cap_rights_contains(&self.0, &other.0) }
     }
 
+    /// Is the given [`Right`] set here?
+    ///
+    /// # Example
+    /// ```
+    /// # use capsicum::{Right, RightsBuilder, FileRights};
+    ///
+    /// let rights = RightsBuilder::new()
+    ///     .allow(Right::Read)
+    ///     .finalize();
+    /// assert!(rights.is_set(Right::Read));
+    /// assert!(!rights.is_set(Right::Write));
+    /// ```
     pub fn is_set(&self, right: Right) -> bool {
         unsafe { libc::__cap_rights_is_set(&self.0 as *const cap_rights_t, right as u64, 0u64) }
     }
 
+    #[deprecated(since = "0.4.0", note = "Unnecessary unless you use FileRights::new")]
+    #[allow(missing_docs)]
     pub fn is_valid(&self) -> bool {
+        self.is_valid_priv()
+    }
+
+    fn is_valid_priv(&self) -> bool {
```

Comment:
Why do we need a `is_valid_priv`?

Reply:
So we can call it in methods like `FileRights::new` without triggering a deprecation warning.  Alternatively, we could just inline it and call the libc method there.